src/plugins/requestAuthPlugin.js

```diff
@@ -107,17 +107,17 @@ export const applyRequestAuthorizer$ = requestAuthorizer$ => request => {
  * Get an anonymous code from the API that can be used to generate an oauth
  * access token
  *
- * @param {Object} config { OAUTH_AUTH_URL, oauth }
+ * @param {Object} server the Hapi app server instance
```

Comment:
I think I'd favor passing just `server.app` instead of the whole `server` instance - the function doesn't need to know about anything other than a few config vars, and the original argument destructuring makes the dependencies explicit.

Not sure if you'd want to do it here, but the `redirect_uri` value is hardcoded in this module and could instead be part of `config` as `oauth_redirect_uri` or something - let me know if you _don't_ want to do that here so that I can make a ticket for doing it later.